.github/workflows/ci.yml

```diff
@@ -48,8 +44,7 @@ jobs:
           path: dist
       - name: Create GitHub release
         run: |
-          export VERSION_PREFIX=$(awk -F '=' '{ print $2 } ' version.properties)
-          VERSION=${VERSION_PREFIX}.$(git rev-list $GITHUB_SHA --count)
+          export VERSION=`echo dist/pc2-*.zip | sed 's#^dist.*pc2-\(.*\).zip*#\1#'`
```

Comment:
I do not understand this line.  It's possible I'm misreading the `sed` expression, but looks to me like this line says "set the value of VERSION to 1".   If that's not what it's saying it would be nice if there was a YAML comment explaining what the line IS doing.   OTOH, if that IS what the line is doing, then why?